aspnet/DateRangePicker/display-format.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: display format
-description: display format
+title: Display format in ASP.NET Web Forms DateRangePicker | Syncfusion
+description: Learn here all about display Format use in Syncfusion ASP.NET Web Forms DateRangePicker control, its elements, and more.
```

Comment:
Update as Learn here all about display format in Syncfusion ASP.NET Web Forms DateRangePicker control, its elements, and more.

Reply:
resolved